src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTestData.cs

```diff
@@ -1423,15 +1423,15 @@ private static IEnumerable<object[]> SerializableObjects()
             if (!PlatformDetection.IsFullFramework/* ? PlatformDetection.IsNetfx471OrNewer : true*/)
```

Comment:
@ViktorHofer  I tried changing this conditional to (PlatformDetection.isnetcore || PlatformDetection.IsNetfx471OrNewer)
but it still gives error that the type is not serialziable

Reply:
it's currently disabled completely for .NET Framework which is correct because we compiled against netfx461 which creates shims that doesn't work well here.